tests/fxl-page-spread_combined/OPS/package.opf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>When rendition:page-spread-right and -left are applied to adjacent spine items, they should be displayed in a synthetic spread.</dc:description>
+    <dc:identifier id="pub-id">fxl-page-spread_combined</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:page-spread- combined</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#fxl-page-spread</meta>
+    <meta property="dcterms:modified">2021-11-24T00:00:00Z</meta>
+    <meta property="rendition:layout">pre-paginated</meta>
+  </metadata>
+  <manifest>
+    <item id="page_001" href="page_001.xhtml" media-type="application/xhtml+xml" />
+    <item id="page_002" href="page_002.xhtml" media-type="application/xhtml+xml" />
+    <item id="page_003" href="page_003.xhtml" media-type="application/xhtml+xml" />
+    <item id="page_004" href="page_004.xhtml" media-type="application/xhtml+xml" />
+    <item id="style"    href="fixed.css" media-type="text/css" />
```

Comment:
I presume the "target" statement for this test is

> When a Reading System encounters two spine items that represent a true spread (i.e., two adjacent spine items with the rendition:page-spread-left and rendition:page-spread-right properties), it SHOULD create the spread with no space between the adjacent pages.

If that is correct, that this is a test for a _SHOULD_ statement. In view of the latest "upgrade" of the reporting mechanism, the following extra metadata statement should be added:

> `<meta property="belongs-to-collection">should</meta>`

(This is to be checked for the other tests, too.)

Reply:
Will add!